Cargo.toml

```diff
@@ -50,6 +50,8 @@ serde_json = { version = "1.0.51", optional = true }
 serde_crate = { version = "1.0.106", features = ["derive"], optional = true, package = "serde" }
 serde_urlencoded = { version = "0.7.0", optional = true}
 serde_qs = { version = "0.8.3", optional = true }
+miette = "2.2.0"
```

Comment:
why this version?

Reply:
ah, so I actually forgot about this until I submitted the PR haha because I originally drafted parts of it in like, September 2021.

I should probably remove it for now and then re-add it in a follow-up PR, but at least this proved that it didn't break anything from this patch